style.css

```diff
@@ -111,3 +112,269 @@ body {
   transition: 1s ease-in-out;
   cursor: pointer;
 }
+
+.works-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+}
+
+.cards {
+  display: flex;
+  flex-direction: column;
+  gap: 88px;
+}
+
+.card {
+  width: 311px;
+  height: 552px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  display: grid;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+}
+
+.card-items {
+  display: grid;
+  place-items: center;
+  padding: 12px;
+  gap: 12px;
+  position: relative;
+  bottom: 3%;
+  margin-left: -15px;
+}
+
+.card-img {
+  width: 311px;
+  height: 220px;
+  border-radius: 8px;
+  margin-top: 11px;
+}
+
+.card-name {
+  width: 295px;
+  height: 36px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.card-props {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  height: 32px;
+  position: relative;
+  bottom: 39px;
+  padding-left: 5px;
+  left: 37px;
+}
+
+.card-prop a {
+  position: relative;
+  right: 64px;
+  height: 16px;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  color: #7a869a;
+  margin-left: 9px;
+  text-decoration: none;
+}
+
+.card-props li:first-child a {
+  color: #344563;
+  text-transform: uppercase;
+}
+
+.card-text {
+  position: relative;
+  bottom: 73px;
+  left: 3px;
+  width: 295px;
+  height: 76px;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.card-languages {
+  display: flex;
+  position: relative;
+  right: 67px;
+  bottom: 103px;
+  justify-content: space-between;
+}
+
+.card-language {
+  font-size: 12px;
+  background-color: #eaebfe;
+  padding: 4px 12px;
+  border-radius: 8px;
+  color: #6070ff;
+  margin-left: 8px;
+}
+
+.card-btn {
+  font-size: 19px;
+  text-align: start;
+  padding: 12px 12px;
+  border: 1px solid #6070ff;
+  width: 131px;
+  border-radius: 8px;
+  margin-bottom: 15px;
+  color: #396df2;
+  background: white;
+  cursor: pointer;
+  position: relative;
+  bottom: 114px;
+  height: 53px;
+  right: 28%;
+}
+
+.about-section {
+  background-color: #fff;
+}
```

Comment:
- [ ] Excellent job building your About me section 🚀. However, please ensure to style the background to look like the Figma design. 🤓 Currently, your design looks very similar, but there is a small curve that is missing. 🔧

Figma design 👇🏽
![figma](https://github.com/josman227/Mobile-Portfolio/assets/116701485/d60a2a10-bebd-41c0-8b0a-1ed7be48ec5b)

Your design 👇🏽
![design](https://github.com/josman227/Mobile-Portfolio/assets/116701485/8ce9306f-73bc-48dc-bac3-d68e405c1add)